api/test/integration/tavern_utils.py

```diff
@@ -508,3 +509,18 @@ def healthcheck_agent_restart(response, agents_list):
     time.sleep(20)
     # Wait for active agent status (up to 25 seconds)
     check_agent_active_status(agents_list)
+
+
+def assert_forwarded_events(response: Response, forwarded_events: str, expected_events: int):
+    """Check if the number forwarded events is equal to the expectede one.
```

Comment:
We have already talked about his several times. Please review the docstrings carefully, use a linter or anything you need to ensure the language is adequate.

There are 2 different failures in this docstring.

number `is` forwarded

`expected` one